subprojects/json.wrap

```diff
@@ -0,0 +1,4 @@
+[wrap-git]
+url = https://github.com/nlohmann/json
```

Comment:
This is available as `meson wrap install nlohmann_json` and `dependency('nlohmann_json')`